extras/data/machinery/generate/EMC.py

```diff
@@ -0,0 +1,40 @@
+from .utils import load, obs_template
+
+
+def dump(src_path, target):
```

Comment:
Can you please check if and where the `target` argument is needed for an experiment? on a first glance this seems to be never used ...

Reply:
BCDMS - no https://github.com/NNPDF/yadism/blob/d367fddcdab9deb4c1a2cd25a5f67c8ea3b60e5b/extras/data/machinery/generate/BCDMS.py#L7
but NMC, yes ... ok I accept